src/components/HeaderComponent.vue

```diff
@@ -0,0 +1,14 @@
+<script setup lang="ts">
+const createTodo = () => {
```

Comment:
Please use **function** instead of arrow functions. Both of them are correct, but with function it will be more clear the code in long term when you want to take difference between variables and functions